Misc/NEWS.d/next/Library/2024-07-22-08-14-04.gh-issue-113785.6B_KNB.rst

```diff
@@ -0,0 +1 @@
+:mod:`csv` now correctly parses numeric fields (when used with :const:`csv.QUOTE_NONNUMERIC`) which start with an escape character.
```

Comment:
and csv.QUOTE_STRINGS.

Reply:
Added in cf05e97ed527587f57dd634d7c2a58422e8e017e